plugins/spammer/cpu_usage.go

```diff
@@ -0,0 +1,107 @@
+package spammer
+
+import (
+	"errors"
+	"io/ioutil"
+	"math/rand"
+	"runtime"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+)
+
+var (
+	CPUUsageTimePerSample = time.Second / 4
+
+	activeWorkerCount = 0
+
+	once sync.Once
+	mu   sync.Mutex
+
+	cpu_last_sum uint64   // previous iteration
+	cpu_last     []uint64 // previous iteration
+
+	cpuUsageResult float64 // current result
+	cpuUsageError  error   // nil || current error
+)
+
+func cpuUsageUpdater() {
+	if runtime.GOOS == "windows" {
+		mu.Lock()
+		defer mu.Unlock()
+		cpuUsageError = errors.New("spammer.maxCPUUsage on Windows is not supported")
```

Comment:
You should panic with this error in the `configure` func of the spammer plugin. No need for this lock, no need for this global error variable.